### PR TITLE
Switch icon hover circle to border from box-shadow and make service-i…

### PR DESCRIPTION
**Pull request by sg00dwin**
…tem display:block to enable word-break
Also
- img max height and width needed to be set
- ` .services-item-icon` flex removed and height set. flex was preventing word-break from functioning

Fixes https://github.com/openshift/origin-web-catalog/issues/201

### CODE REVIEW

**Comment by sg00dwin** [comment]
@rhamilto PTAL
cc @spadgett

**Comment by dtaylor113** [comment]
It would be great to see a screenshot :-)

File: src/styles/services-view.less [hunk 1]
```diff
@@ -43,19 +43,17 @@
     }
 
     .services-item-icon {
-      border-radius: 50%;
-      height: 90px;
-      width: 90px;
       &:after {
         border-radius: 100px;
-        box-shadow: 0 0 0 2px transparent;
+        border: 2px solid
```
Comment (rhamilto) [inline src/styles/services-view.less]: Nit:  alpha

**Comment by sg00dwin** [comment]
@spadgett @rhamilto Updated pr for review

Added gradient fade and overview hidden to handle long names.

screen shot 2017-05-04 at 2 21 32 pm
screen shot 2017-05-04 at 2 21 11 pm

**Comment by sg00dwin** [comment]
pr rebased

**Comment by beanh66** [comment]
@sg00dwin @rhamilto I think we should use an ellipses when text gets cut off. In this case we would not use the fade, we would just show the ... after three lines of text and then show the full name in a tooltip on hover. This would be in line with PF and we should also be consistent with what we decide here: https://github.com/openshift/origin-web-catalog/pull/214 @serenamarie125

**Comment by spadgett** [comment]
@beanh66 We can only do this if we decide to truncate to a certain number of characters instead of cleanly at the end of the 3rd line. There's no good way that I know in CSS to use an ellipsis to truncate a multiline string. @rhamilto keep me honest here

This will be a bigger problem for #214 where we can't truncate by characters since it will break any links that get truncated the middle. We'd have to remove the link highlighting, and it also means we might truncate in the middle of a line.

**Comment by rhamilto** [comment]
> @beanh66 We can only do this if we decide to truncate to a certain number of characters instead of cleanly at the end of the 3rd line. There's no good way that I know in CSS to use an ellipsis to truncate a multiline string. @rhamilto keep me honest here

To my knowledge, there is no cross-browser compatible solution as described, which is why we went with the fade out effect here and in in #214.  :(

**Comment by beanh66** [comment]
@spadgett @rhamilto Can we cut these titles off at 40 characters then show the ellipses? We would still allow users to see the full title on hover. Thoughts @ncameronbritt @serenamarie125?

**Comment by sg00dwin** [comment]
@spadgett updated with overflow gradient removed.

**Comment by spadgett** [comment]
@serenamarie125 @beanh66 @rhamilto I asked @sg00dwin to remove the gradient for now and add the ellipsis in a follow-on. We need to find a good way to do that both here and the overlay panel, but unfortunately it's a little tricky.

**Comment by spadgett** [comment]
@rhamilto Can you take another look?

**Comment by rhamilto** [comment]
[merge]

**Comment by serenamarie125** [comment]
agreed @spadgett I think it's also worth us discussing best practices, and making sure our recommendations are documented appropriately.  I know that we will still need to address, but if we document best practices, hopefully these edge cases will be minimized.

**Pull request merged**